vg/vgpdf/vgpdf.go

```diff
@@ -249,6 +249,17 @@ func (c *Canvas) pdfPath(path vg.Path, style string) {
 			c.doc.LineTo(c.pdfPoint(comp.Pos))
 		case vg.ArcComp:
 			c.arc(comp, style)
+		case vg.CurveComp:
+			if len(comp.Control) > 0 {
+				cx, cy := c.pdfPoint(comp.Control[0])
+				px, py := c.pdfPoint(comp.Pos)
+				if len(comp.Control) == 1 {
```

Comment:
ditto.

Reply:
I interpret "I would perhaps" as to wait for another opinion. Otherwise please tell me, and I do it.

Reply:
yep. let's wait for @kortschak to chime in.

Reply:
Yes, this improves readability.

Reply:
Ok, it's done.